source/speech.py

```diff
@@ -408,23 +408,28 @@ def splitTextIndentation(text):
 	return RE_INDENTATION_SPLIT.match(text).groups()
 
 RE_INDENTATION_CONVERT = re.compile(r"(?P<char>\s)(?P=char)*", re.UNICODE)
+BASE_FREQUENCY = 220 #One octave below middle A.
+TONE_DURATION = 80 #Milleseconds
+MAX_SPACES = 72

```

Comment:
nit: Please prefix these so it's clear they're related to indentation when looking at the namespace. Perhaps an `INDTONE_` prefix; e.g. `INDTONE_MAX_SPACES`?